roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -451,7 +451,7 @@
     "os_compute_api:os-server-usage:discoverable": "@",
     "os_compute_api:os-server-groups": "rule:admin_or_owner",
     "os_compute_api:os-server-groups:discoverable": "@",
-    "os_compute_api:os-services": "rule:admin_api",
+    "os_compute_api:os-services": "rule:admin_api or role:cloud_admin",
```

Comment:
What error we see? cloud_admin do not have access to os-services. We should not make this change as this will allow cloud_admin to disable services.